docs/bucket/notifications/README.md

```diff
@@ -1,19 +1,23 @@
 # Minio Bucket Notification Guide [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
 
-Changes in a bucket, such as object uploads and removal, can be monitored using bucket event notification
-mechanism and can be published to the following targets:
-
-| Notification Targets|
-|:---|
-| [`AMQP`](#AMQP) |
-| [`MQTT`](#MQTT) |
-| [`Elasticsearch`](#Elasticsearch) |
-| [`Redis`](#Redis) |
-| [`NATS`](#NATS) |
-| [`PostgreSQL`](#PostgreSQL) |
-| [`MySQL`](#MySQL) |
-| [`Apache Kafka`](#apache-kafka) |
-| [`Webhooks`](#webhooks) |
+Events occurring on objects in a bucket can be monitored using bucket event notifications. Event types supported by Minio server are
+
+| Event Types | | |
+|:---------------------------|--------------------------------------------|-------------------------|
+| `s3:ObjectCreated:Put`     | `s3:ObjectCreated:CompleteMultipartUpload` | `s3:ObjectCreated:Head` |
```

Comment:
We also support s3:ObjectRemoved as well, and also that we extended the scope here and added s3:ObjectAccessed for Get and Head.

Reply:
Ah yes, that was typo, fixed now.